### DISCUSSION TITLE
Unbind Ctrl-k (and Ctrl-j)

### DISCUSSION THREAD

**Discussion by metaturso** [junegunn/fzf#3824] [Q&A] [answered]
When using Emacs-like keybindings `Ctrl-k` should kill the line forward, i.e. cut from the cursor up to the end of the line. Right now it moves the selection up instead.

How do I unbind `Ctrl-k`, and `Ctrl-j` while I'm at it, and let readline handle them as intended?

**Comment by LangLangBart** [comment, marked as answer]
I think your best option would be to remap the keys in your `FZF_DEFAULT_OPTS` or `FZF_DEFAULT_OPTS_FILE`  to change how `fzf` responds to them. 



| key | default action in `fzf` |
|--------|--------|
| <kbd>⌃ Control</kbd> + <kbd>k</kbd> | up |
| <kbd>⌃ Control</kbd> + <kbd>j</kbd> | down | 

Here's an example of how to remap keys, all available actions are listed in the `fzf` man page.

```sh
export FZF_DEFAULT_OPTS="--bind=ctrl-j:accept,ctrl-k:kill-line"
```

**Reply by metaturso** [reply]
That's pretty neat actually. Thanks for the example.
